docs/products/valkey/concepts/read-replica.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Aiven for Valkey read replica
+sidebar_label: Valkey read replica
+early: true
+---
+
+import Button from "@site/src/components/Buttons";
+import GridContainer from "@site/src/components/GridContainer";
+
+Aiven for Valkey read replica replicates data from a primary cluster to a replica cluster across different DNS zones, clouds, or regions, enhancing data availability and supporting disaster recovery.
```

Comment:
i would replace `cluster` with `service`. Technically, the primary service can be an HA cluster, but it could also be a single node. And the secondary service can only be single node service.